projects/standard-rulesets/src/breaking-changes/index.ts

```diff
@@ -125,13 +115,35 @@ export class BreakingChangesRuleset extends Ruleset<BreakingChangesRules> {
     return new BreakingChangesRuleset(constructorConfig);
   }
 
-  constructor(
-    config: Omit<RulesetConfig<BreakingChangesRules>, 'name' | 'rules'> = {}
-  ) {
+  constructor(config: ConstructorConfig = {}) {
+    const breakingChangesRules = [
```

Comment:
rulesets options were bleeding between breaking change instances - since we're setting sev on the rules we need each breaking change instance to have it's own rule instances